Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.h

```diff
@@ -127,6 +131,8 @@ class ModelProcessModelPlayerProxy final
     float m_pitch { 0 };
     float m_yaw { 0 };
     REAnimationPlaybackToken m_animationPlaybackToken { kInvalidAnimationToken };
+
+    RESRT m_srt;
```

Comment:
Please expand the "srt" acronym (I have no idea what it means).

Reply:
Good one. Added more comments as well.
```
    RESRT m_transformSRT; // SRT=Scaling/Rotation/Translation. This is stricter than a WebCore::TransformationMatrix.
```